modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -73,7 +73,6 @@ CV__DNN_INLINE_NS_BEGIN
         //! OpenCV is built with Intel OpenVINO or
         //! DNN_BACKEND_OPENCV otherwise.
         DNN_BACKEND_DEFAULT = 0,
-        DNN_BACKEND_HALIDE,
```

Comment:
Please assign values to other enumerations so removing `DNN_BACKEND_HALIDE` won't break them.

Reply:
I set `DNN_BACKEND_INFERENCE_ENGINE = 2,` to keep compatibility.